sdk/src/utils/package.json

```diff
@@ -0,0 +1,3 @@
+{
+    "type": "module"
+}
```

Comment:
Why do we need these (this and src/types/package.json)?

Reply:
for the pnpm workspaces setup, the presence of each `package.json` indicates a new submodule within the workspace